### ISSUE TITLE
From version 0.4.0 this package started to override my user.sessions.create method

### ISSUE THREAD

**Issue by emish89** [ruby/securerandom#35]
Hi, 

This one is quite tricky and it took some time to analyze.

After a "bundle u" in my project, this dependency updated from `0.3.2` to `0.4.0`.

In version `0.4.0`, this package started overriding my `user.sessions.create` method and I don't understand why.

When I call my `user.sessions.create` method, I now get a strange :

```
full_message': wrong number of arguments (given 2, expected 0..1) (ArgumentError)
```

It is 100% due to the version upgrade of this package but I can't easily understand why.

Thanks a lot for the help

cc @hsbt

**Comment by fsmanuel** [comment]
I'm having the same issue!

**Comment by alejandrobabio** [comment]
After removing `lib/random/formatter.rb` it starts to use the ruby version of `random/formatter.rb#alphanumeric` which expect `0..1` parameter instead of 2.

```
     ArgumentError:
       wrong number of arguments (given 2, expected 0..1)
     # /Users/ale/.asdf/installs/ruby/3.2.1/lib/ruby/3.2.0/random/formatter.rb:241:in `alphanumeric'
     # /Users/ale/.asdf/installs/ruby/3.2.1/lib/ruby/gems/3.2.0/gems/activesupport-7.2.2.1/lib/active_support/core_ext/securerandom.rb:20:in `base58'
```

**Comment by hsbt** [comment]
I fixed this at #36 and released 0.4.1 with it.
